src/components/Transfer.tsx

```diff
@@ -111,20 +119,20 @@ function Transfer() {
         />
       </Box>
       <Receiver />
-      <ButtonSubmit disabled={isButtonDisabled() || !!transferAmountError} onClick={sendLaneMessage}>
+      <ButtonSubmit disabled={!transactionReadyToExecute || amountNotCorrect} onClick={sendLaneMessage}>
         Send bridge transfer from {sourceChainDetails.chain} to {targetChainDetails.chain}
       </ButtonSubmit>
       {amountNotCorrect ? (
         <Alert severity="error">
           Account&apos;s amount (including fees: {estimatedFee}) is not enough for this transaction.
         </Alert>
-      ) : estimatedFeeLoading ? (
+      ) : estimatedFeeLoading && transferAmount && receiverAddress ? (
         <Typography variant="body1" color="secondary">
           Estimated source Fee loading...
         </Typography>
       ) : (
         <Typography variant="body1" color="secondary">
-          {receiverAddress && estimatedFee && `Estimated source Fee: ${estimatedFee}`}
+          {receiverAddress && estimatedFee && transferAmount && `Estimated source Fee: ${estimatedFee}`}
```

Comment:
i don't really like this validations for showing the `estimatedFee` ... i suspect merging this PR with the PR #218  will solve the need for so many checks.